pallets/network-score/src/tests.rs

```diff
@@ -667,3 +667,103 @@ fn rating_identifier_not_found_test() {
 		);
 	});
 }
+
+#[test]
+fn reference_identifier_not_found_test() {
+	let creator = DID_00.clone();
+	let author = ACCOUNT_00.clone();
+	let message_id = BoundedVec::try_from([82u8; 10].to_vec()).unwrap();
+	let message_id_revise = BoundedVec::try_from([75u8; 10].to_vec()).unwrap();
+	let message_id_revoke = BoundedVec::try_from([84u8; 10].to_vec()).unwrap();
+	let entity_id = BoundedVec::try_from([73u8; 10].to_vec()).unwrap();
+	let provider_id = BoundedVec::try_from([74u8; 10].to_vec()).unwrap();
+
+	let entry = RatingInputEntryOf::<Test> {
+		entity_id: entity_id.clone(),
+		provider_id: provider_id.clone(),
+		total_encoded_rating: 250u64,
+		count_of_txn: 7u64,
+		rating_type: RatingTypeOf::Overall,
+		provider_did: creator.clone(),
+	};
+
+	let entry_digest =
+		<Test as frame_system::Config>::Hashing::hash(&[&entry.encode()[..]].concat()[..]);
+
+	let raw_space = [2u8; 256].to_vec();
+	let space_digest = <Test as frame_system::Config>::Hashing::hash(&raw_space.encode()[..]);
+	let space_id_digest = <Test as frame_system::Config>::Hashing::hash(
+		&[&space_digest.encode()[..], &creator.encode()[..]].concat()[..],
+	);
+	let space_id: SpaceIdOf = generate_space_id::<Test>(&space_id_digest);
+	let auth_digest = <Test as frame_system::Config>::Hashing::hash(
+		&[&space_id.encode()[..], &creator.encode()[..], &creator.encode()[..]].concat()[..],
```

Comment:
```
let auth_digest = <Test as frame_system::Config>::Hashing::hash(
		&[&space_id.encode()[..], &creator.encode()[..]].concat()[..],
	);
```
Correct this declaration of auth_digest , or else it will give `AuthorizationNotFound` Error

Reply:
In all other functions too
```
let auth_digest = <Test as frame_system::Config>::Hashing::hash(
		&[&space_id.encode()[..], &creator.encode()[..], &creator.encode()[..]].concat()[..],
	);

``` 
this is used

on using the below snippet does not meet the requirements
```
let auth_digest = <Test as frame_system::Config>::Hashing::hash(
		&[&space_id.encode()[..], &creator.encode()[..]].concat()[..],
	);
```

Reply:
where is the first code snippet used elsewhere , I got this 





![Screenshot 2024-09-16 210759](https://github.com/user-attachments/assets/f42b4560-3d3d-4809-ad04-8c28d72ffa1a)

Reply:
Sir my test is running fine, because if we give only two requirements as mentioned by you , the length would be short then it would give error. but the snippet which I have used is running error-free

Reply:
> Sir my test is running fine, because if we give only two requirements as mentioned by you , the length would be short then it would give error. but the snippet which I have used is running error-free



> Sir my test is running fine, because if we give only two requirements as mentioned by you , the length would be short then it would give error. but the snippet which I have used is running error-free



> Sir my test is running fine, because if we give only two requirements as mentioned by you , the length would be short then it would give error. but the snippet which I have used is running error-free

Can you run this command and check if the tests are running error-free
```
cargo test --release --locked --features=runtime-benchmarks --no-fail-fast --verbose --color always -p pallet-network-score --lib
```

Reply:
by this command also its working fine 
Screenshot -
![image](https://github.com/user-attachments/assets/7b08d890-8a39-4cb2-8ace-12ed383b7adb)